### PR TITLE
Add a betamocks route for Appeals NOD

### PR DESCRIPTION
**Pull request by charleystran**
This PR adds a betamocks path for hitting the NOD endpoint on caseflow so we can record and use the new responses.

### CODE REVIEW

**Review by templeman15 — APPROVED** [review]
First time seeing this file, but looks good comparatively to other places in the file.

File: config/betamocks/services_config.yml [hunk 1]
```diff
@@ -470,6 +470,12 @@
     :cache_multiple_responses:
       :uid_location: header
       :uid_locator: X-VA-SSN
+  - :method: :get
+    :path: "/api/v3/decision_reviews/appeals/contestable_issues"
+    :file_path: "modules_appeals_api/decision_reviews/contestable_issues"
```
Comment (edmangimelli) [inline config/betamocks/services_config.yml:475]:
```suggestion
    :file_path: "modules_appeals_api/notice_of_disagreements/contestable_issues"
```

**Review by edmangimelli — CHANGES_REQUESTED** [review]
i think you might have a typo

**Review by edmangimelli — APPROVED** [review]
😍

**Pull request merged**